src/styles/blocks/stars--1.css

```diff
@@ -0,0 +1,44 @@
+.stars--1 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
+}
+
+.stars--1 .stars__star:nth-child(2) {
+  display: block;
+  background-image: url(../../images/star.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 4px;
```

Comment:
Avoid repeating the same styles. The `.stars__star` elements share common styles such as 'display', 'background-image', 'width', 'height', and 'margin-left'. Extract these common properties to a shared class and apply individual differences using additional classes or modifiers.